### PR TITLE
[PFCMP-0011] Empty View

### LINKED ISSUE #111: [PFCMP-0011] Empty View
### Priority 
- [x] Low
- [ ] Mid
- [ ] High

### Links
Name | Link
-|-
User Story | [link]()
Figma | [link]()

### Completion
Expected | Time
-|-
Expected Start | 16-11-2020
Expected End | 17-11-2020

Success Criteria:
- Create Component for empty view
- Define location for programatic use
- View Hierarchy:
+ Image 
+ Description Label

### PR DESCRIPTION
**Pull request by SetyaG**
Closes #111 
***
## Description
Create the empty view component to be used when there's no data found

## Related Links
List related PRs against other branches:

List | link
------ | ------
User Story | [link]()
Figma | [link]()
Issue | [link](#111)

## Todos
- [X] UI
- [ ] Coordinator
- [ ] Network
- [ ] Tests
- [ ] Documentation

## Screenshot / Video
Screenshot consisting of changes made on this branch.

### UI Changes
UI related changes will be shown here (can be either screenshot or video format)

### Network Status
Network related changes will be shown here (screenshot of request status)

### Test Coverage
Test coverage result for designated changes will be shown here (screenshot of coverage)

## Notes
Debt: 
- Elements somehow still don't exactly positioned at the center

### CODE REVIEW

**Comment by inxudianz** [comment]
Please provide picture by testing it a random viewController

**Comment by SetyaG** [comment]
Screen Shot 2020-11-18 at 19 23 51

**Comment by SetyaG** [comment]
The sdasda sentence is the text, the image should be above but somehow it doesn't appear

File: Component/Component/View/EmptyView/FamiliEmptyView.swift [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    @IBInspectable var labelText: String = "" {
+        didSet {
+            emptyViewLabel.text = labelText
+            setupLabel()
+        }
+    }
+    
+    @IBInspectable var imageName: String = "" {
+        didSet {
+            guard let image = UIImage(named: imageName) else { return }
```
Comment (inxudianz) [inline Component/Component/View/EmptyView/FamiliEmptyView.swift]: use UIImage(named: , in: ,compatibleWith:) initialization to make image visible

**Review by inxudianz — CHANGES_REQUESTED** [review]

File: Component/Component/View/EmptyView/FamiliEmptyView.swift (hunk 1 above)
Comment (SetyaG) [reply Component/Component/View/EmptyView/FamiliEmptyView.swift]: updated

**Comment by SetyaG** [comment]
Screen Shot 2020-11-20 at 14 19 42

Image now can be shown

**Review by inxudianz — APPROVED** [review]

**Pull request merged**